bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/ApiBridgeHandler.java

```diff
@@ -215,8 +215,9 @@ private void startAuthorizationFlow() {
         GrantServlet servlet = new GrantServlet(this, httpService);
         servlet.startListening();
         grantServlet = Optional.of(servlet);
-        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
-                "@text/conf-error-grant-needed [ \"%s\" ]".formatted(servlet.getPath()));
+        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING,
```

Comment:
I don't think this status is correct, at least not according to the documentation:
https://www.openhab.org/docs/concepts/things.html#status-details

So I guess we would either need to keep the existing thing status detail as `CONFIGURATION_ERROR` or we would need to redefine/expand the meaning of `CONFIGURATION_PENDING`?

@openhab/add-ons-maintainers, @openhab/core-maintainers - WDYT, would it make sense to expand `CONFIGURATION_PENDING` to cover a case like this as well, or perhaps create a new thing status detail for it?

Reply:
I must admit I have absolutely no idea how the netatmo binding works and how to configure it.

In other binding with similar "click here to do something (login, or enable access, etc) before I'm happy", e.g. amazonechocontrol, they use OFFLINE+CONFIGURATION_PENDING. 

The table you linked does not state that this is a valid combination. According to the table, CONFIGURATION_PENDING needs to be coupled with ONLINE. If this was the case, then those other bindings are wrong and they need to change to OFFLINE+CONFIGURATION_ERROR?

I'm happy to go either way and adjust all the other bindings + the new webui code.

Reply:
OK I've reverted it back to CONFIGURATION_ERROR. I will also make changes in the other bindings to match this.

Reply:
I found many instances of OFFLINE+CONFIGURATION_PENDING in other bindings. Do we want to go through each and every one of them and correct the combinations? It's not as simple as changing them to OFFLINE+CONFIGURATION_ERROR. In some cases, they need to change to INITIALIZING+NONE but this needs to be evaluated on a case-by-case basis.

I think I'll leave them be for now.

Reply:
> I guess this PR now depends on another change in the webui repo, right?

Yes, but it's a chicken or egg question. Webui could argue that it's waiting for this pr.

Reply:
If this is merged first, the UI will look strange right, i.e. it will display something like "http(s)://<YOUROPENHAB>:<YOURPORT>" without any substitution, correct?

I don't think webui is waiting for anything: with openhab/openhab-webui#2996 it now has a feature that is currently unused (but doesn't do any harm). When that feature is updated to support substitution for `CONFIGURATION_ERROR`, that will still be the case (no usage, no harm). Then, merging this PR will make use of this new feature. Or maybe I'm missing something?

Reply:
You're correct. Amazon echo and one other bindings were already in this format even before the original webui pr. Anyway, waiting is fine.

Reply:
The webui pr has been merged